### PR TITLE
BAN-142: Bringed tests back running transforms with babel-jest

### PR DESCRIPTION
**Pull request by f-jauregui**
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/BAN-142)

#### Description
As per JIRA ticket

#### ChangeLOG

- [ ] I have added all notable deployment/development environment (onprem!) changes to [CHANGELOG.md](https://github.com/reciprocity/zen-ui/blob/main/README.md)

#### Useful links
[Zen UI readme](https://github.com/reciprocity/zen-ui/blob/main/README.md)

[PR review guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/pull_request_reviews.md)

[Git commit guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/commit_guidelines.md)

### CODE REVIEW

File: stencil.config.ts [hunk 1]
```diff
@@ -21,4 +21,10 @@ export const config: Config = {
     },
   ],
   plugins: [sass()],
+  testing: {
+    transform: {
+      '^.+\\.js?$': 'babel-jest',
+    },
+    transformIgnorePatterns: ['node_modules/(?!query-selector-shadow-dom/)'],
```
Comment (mihaelpalfi) [inline stencil.config.ts:28]: Does this mean that we will always have to add tests imported libraries to the ignore patterns?

**Review by mihaelpalfi — APPROVED** [review]
Tests are running again. Good work! Just one thing that gives me second thoughts is this always adding libraries to ignore patterns in the config.

File: stencil.config.ts (hunk 1 above)
Comment (chavlji) [reply stencil.config.ts:28]: I'm asking myself the same thing :( Couldn't we just keep transpile for all js files?

**Review by chavlji — APPROVED** [review]
Great job Fermin!

File: stencil.config.ts (hunk 1 above)
Comment (f-jauregui) [reply stencil.config.ts:28]: Not all of them, just the ones that are using ESM.
I could not find other way. We can try this https://nodejs.org/api/vm.html#vm_class_vm_module but it's still in experimental phase, that's why Jest doesn't support it.

Comment (chavlji) [reply stencil.config.ts:28]: No problem. It's ok for now. We can try to improve it later. It's great to have tests back! 🚢 it.

Comment (f-jauregui) [reply stencil.config.ts:28]: Cool. And just to clarify why Jest doesn't support this we'all can see it here https://jestjs.io/docs/en/ecmascript-modules at the beginning.

**Pull request merged**